content/en/examples/controllers/job-backoff-limit-per-index-example.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: job-backoff-limit-per-index-example
+spec:
+  completions: 10
+  parallelism: 3
+  completionMode: Indexed  # required for the feature
+  backoffLimitPerIndex: 1  # maximal number of failures per index
+  maxFailedIndexes: 5      # maximal number of failed indexes before terminating the Job execution
+  template:
+    spec:
+      restartPolicy: Never # required for the feature
+      containers:
+      - name: example
+        image: python
+        command:
+        - python3
+        - -c
+        - |
+          import os, sys
+          print("Hello world")
+          if int(os.environ.get("JOB_COMPLETION_INDEX")) % 2 == 0:
+            sys.exit(1)
```

Comment:
Consider adding a comment to explain that this example Job always **fails** (it looks to me that it does).

Reply:
Correct, added a comment. Though I'm thinking about a better example. Let me know if you have an idea. Some example could probably go to Tasks also, but I'm wondering a good example idea.

Reply:
It'd be nice to have a Job(s) tutorial within https://kubernetes.io/docs/tutorials/#stateless-applications

Something for a separate PR!

Reply:
Ok, so I see you suggest the tutorial for later, but should we include a task (say "Execute all indexes") in this PR? I can propose something.

Reply:
For alpha, no need. For beta, a page is handy if people are genuinely likely to want to perform that task outside of a learning context.

For example, _Enforce Pod Security Standards with Namespace Labels_ is a plausible task, whereas _Explore Termination Behavior for Pods And Their Endpoints_ is clearly a tutorial.

Reply:
Even if it is still alpha having some more step-by-step examples might be useful. Especially since there is also the new FailIndex action in pod failure policy. I will propose an update to this yet.

Reply:
@sftim I've added the proposal for the tasks page. PTAL

Reply:
I think the current contents of the task are an overkill for the feature.

Reply:
Resolving as the original comment is addressed. The side topic of a new task is discussed in a separate thread: https://github.com/kubernetes/website/pull/41921#discussion_r1275256874